browser/src/slideshow/engine/SlideShowNavigator.ts

```diff
@@ -194,8 +194,11 @@ class SlideShowNavigator {
 				', bSkipTransition: ' +
 				bSkipTransition,
 		);
-		if (nNewSlide === undefined || nNewSlide < 0) return;
-		if (nNewSlide >= this.theMetaPres.numberOfSlides) {
+		if (
+			nNewSlide === undefined ||
+			nNewSlide < 0 ||
+			nNewSlide >= this.theMetaPres.numberOfSlides
+		) {
 			this.currentSlide = nNewSlide;
 			const force = nNewSlide > this.theMetaPres.numberOfSlides;
 			if (force) this.quit();
```

Comment:
It's annoying that it's not possible to add a comment to any code line.
Anyhow, I think we should change in `if (!slideAvailable)` (~line 225) `this.currentSlide !== undefined` to `this.currentSlide !== undefined && this.currentSlide !== nNewSlide).
I suspect it can cause an infinite recursion when `this.currentSlide === nNewSlide === 0 and the first slide is hidden or no more available.
Or maybe we could check at `displaySlide` beginning and return immediately in case `this.currentSlide === nNewSlide`.